app/lib/proof_provider/keybase.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class ProofProvider::Keybase
+  BASE_URL = 'https://keybase.io'
+
+  class Error < StandardError; end
+
+  class ExpectedProofLiveError < Error; end
+
+  class UnexpectedResponseError < Error; end
+
+  def initialize(proof = nil)
+    @proof = proof
+  end
+
+  def serializer_class
+    ProofProvider::Keybase::Serializer
+  end
+
+  def worker_class
+    ProofProvider::Keybase::Worker
+  end
```

Comment:
I feel like this should be derived rather than explicit? AKA `ProofProvider.const_get(self.class.name)::Worker`

Reply:
I was considering using metaprogramming, and this might be a fair critique, but since it's just two methods that do this, I opted to just write them out. I am generally more comfortable with less metaprogramming involved because it simplifies finding errors and searching code for usage.

Reply:
Understandable.